extensions/blocks/block-helpers/block-helpers.ts

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
We have an `extensions/shared/` directory where we place things like this:
`extensions/shared/block-category.js`

Do you think we could continue to use that instead of the new `block-helpers` directory?

Reply:
Ah, thanks for the suggestion. I think it does makes sense to move it there :+1: